bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBatchExecutor.java

```diff
@@ -162,7 +186,7 @@ public void testShutdownService() throws Exception {
     service.shutdown();
     service.awaitTermination(1000, TimeUnit.MILLISECONDS);
     try {
-      underTest.batch(Arrays.asList(new Put(EMPTY_KEY)));
+      createExecutor(DEFAULT_OPTIONS).batch(Arrays.asList(new Put(EMPTY_KEY)));
```

Comment:
I'm not sure what you mean.  Aren't we already doing that?  Do you mean randomPut()?